pkg/scalers/aws_sigv4.go

```diff
@@ -0,0 +1,111 @@
+package scalers
```

Comment:
we have a package for aws related stuff, we shoudl probably move it there: `scalers/aws`

Reply:
This is still pending

Reply:
There is a problem with ScalersConfig circular dependency, we should refactor the package: https://github.com/kedacore/keda/issues/5379